scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -253,6 +253,8 @@ main (int argc, char *argv[])
   double simulationTime = 10;                   /* Simulation time in seconds. */
   bool pcapTracing = false;                     /* PCAP Tracing is enabled or not. */
   std::map<std::string, std::string> tcpVariants; /* List of the tcp Variants */
+  uint16_t tos = 0;                             /* Select AC_BE as default AC */
+  std::vector<uint8_t> tosValues = {0x70, 0x28, 0xb8, 0xc0}; /* AC_BE, AC_BK, AC_VI, AC_VO */
```

Comment:
If possible, give reference to where these values can be found. makes debugging easier